apps/wing/src/cli.ts

```diff
@@ -182,6 +186,12 @@ async function main() {
       DEFAULT_PLATFORM
     )
     .option("-r, --rootId <rootId>", "App root id")
+    .option(
+      "--value <value>",
+      "Platform-specific values in the form KEY=VALUE or yaml file",
+      addValue,
+      []
+    )
```

Comment:
Since this is now variadic what will happen if the user provides 
```sh
wing compile app.main.w -v a.yml -v b.yml
```

Reply:
BOOM 
![RunningExplosionGIF](https://github.com/winglang/wing/assets/67694075/108fcda8-011d-4b2a-bf24-0eadc02d960d)


I need to fix this 🤯